redirect/routes.php

```diff
@@ -110,12 +110,20 @@ class Route {
         ],
         'tmmi'         => [
             'description' => 'Technik der Mensch inen Interaktion [Not Found]',
-            'moodle_id'   => '-1'
+            'moodle_id'   => '6662'
         ],
         'mgl2'         => [
             'description' => 'Mathematische Grundlagen 2 [Lorencka]',
             'moodle_id'   => '4130'
         ],
+        'ffi_mathe2'   => [
```

Comment:
umbenennung zu ffi_mg2 um FFI Stundenplan konform zu sein
```suggestion
        'ffi_mg2'   => [
```